app/src/components/Scheduling/ContactReviewStyles.js

```diff
@@ -0,0 +1,48 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles({
+  root: {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    margin: '1rem',
+  },
+  input: {
+    display: 'inline-flex',
+    flexDirection: 'row',
+    alignItems: 'center',
```

Comment:
To address your 'hacky' solution, add `position: relative;` here, and 

```js
right: 0,
top: 13,
```
To the editButton class on line 22. (It's a javascript object so you can enter integer values as opposed to `right: '0px';`)

Reply:
Thank you so much!!